pkg/apiserver/storage/testing/watcher_tests.go

```diff
@@ -44,11 +44,11 @@ func RunTestWatch(ctx context.Context, t *testing.T, store storage.Interface) {
 // - update that gets filtered should trigger Deleted event
 func testWatch(ctx context.Context, t *testing.T, store storage.Interface, recursive bool) {
 	basePod := &example.Pod{
-		ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+		ObjectMeta: metav1.ObjectMeta{Name: "foo", UID: "123"},
```

Comment:
i don't think these should need to be modified to pass 🤔

Reply:
they actually fail without it 😞